### PR TITLE
Change Dockerfile to run with a non-root user

### PR DESCRIPTION
**Pull request by Kaitou786**

### CODE REVIEW

**Comment by Kaitou786** [comment]
Using the prebuilt node-image-alpine reduced the overall image size significantly, it is less than half the size it was before :)

**Review by Ginxo — APPROVED** [review]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -1,18 +1,15 @@
-FROM registry.access.redhat.com/ubi8/ubi
```
Comment (ricardozanini) [inline Dockerfile:1]: Why not using Red Hat's image?

**Comment by Kaitou786** [comment]
Changed the image with one from the RedHat registry, the user in image is `default` with id `1001`. The only thing is the image is lot larger than earlier :/

**Review by ricardozanini — APPROVED** [review]

**Pull request merged**